docs/Sonarr/Sonarr-setup-custom-formats-Anime.md

```diff
@@ -0,0 +1,86 @@
+# How to setup Custom Formats (Anime)
+
+!!! attention
+    You must be running Sonarr V4 to be able to use this setup
+
+!!! note
+    This guide is created and maintained by "Update Later"
```

Comment:
Update later ?

Reply:
Placeholder haha hadn't filled it out yet